README.md

```diff
@@ -1,18 +1,16 @@
 # authenticate-with-gh-package-registries
 
-This action will scan the file provided in the `file-with-substitutions` argument for Octopus variable substitution syntax `#{VariableName}`.  If it finds a matching variable in the `variables-file`, it will replace the template with the actual value.
-
-The variable value and keys can all be placed inside of one file.  The substitution has to be performed one file at a time though.
+This action will scan the file(s) provided in the `file-with-substitutions` argument for Octopus variable substitution syntax `#{VariableName}`.  If it finds a matching variable in the `variables-file` or in the environment variables, it will replace the template with the actual value.
```

Comment:
I think I wrote this summary initially and now I think it needs some clarification...
```suggestion
This action will scan the file(s) provided in the `files-with-substitutions` argument for Octopus variable substitution syntax `#{VariableName}`.  If the files contain any `#{Variables}` that match an item in the `variables-file` or environment variables, it will replace the template with the actual value.
```